common/views/components/SearchToolbar/SearchToolbar.js

```diff
@@ -117,26 +124,47 @@ const SearchToolbar = () => {
               ))}
             </div>
           </Section>
-        </div>
-        <a
-          href="https://docs.wellcomecollection.org/catalogue/search_relevance/tests"
-          className={classNames({
-            flex: true,
-          })}
-        >
-          <Space
-            h={{
-              size: 'xs',
-              properties: ['margin-right'],
-            }}
-            v={{
-              size: 'xs',
-              properties: ['padding-top'],
-            }}
+          <Section
+            className={classNames({
+              flex: true,
+            })}
+          >
+            <label>
+              <Input
+                type="checkbox"
+                checked={unfilteredSearchResultsState}
+                name="unfilteredSearchResults"
+                onChange={event => {
+                  const checked = event.currentTarget.checked;
+                  document.cookie = `toggle_unfilteredSearchResults=${checked}; path=/; max-age=31536000; SameSite=Lax;`;
```

Comment:
Will this wipe out any pre-existing cookies, or does it append?

Reply:
ignore me, I've just had a look

Reply:
Na, strange implementation that, but the web being backwards compatible and all that.